lib/dbDriver/mysql.js

```diff
@@ -0,0 +1,16 @@
+const connection = require('../../knexfile');
+// eslint-disable-next-line import/order
+const knex = require('knex')(connection.development);
+
+class MySQLDriver {
+  // eslint-disable-next-line class-methods-use-this
+  addLink(project, label, url) {
+    return knex('project_links')
+      .insert({ project_id: project, label, url })
+      .catch((e) => {
+        console.log(e);
```

Comment:
We will never see this `console.log` in production. Let's let the system fail by not catching errors like this until we have a logging/diagnostics approach in place that will work in production. cc @nrambeck